### PR TITLE
Prepare for v0.16.0 release

### PR DESCRIPTION
**Pull request by justinvp**

### CODE REVIEW

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ### Improvements
 
-- Classloader: now supports dynamic exclusions of packages from system property.
+- Add support for attaching debuggers
 
 ### Bug Fixes
 
-- Fix HTML escaping after `{@literal ...}` blocks in Javadocs
-- Fix issue with direct dependencies causing IndexOutOfBoundsException
```
Comment (Frassle) [inline CHANGELOG_PENDING.md:8]: Doesn't look like this line is in CHANGELOG.md

Comment (justinvp) [reply CHANGELOG_PENDING.md:8]: Good catch. Fixed up CHANGELOG_PENDING.md and CHANGELOG.md.

**Review by Frassle — APPROVED** [review]

File: CHANGELOG_PENDING.md (hunk 1 above)
Comment (Frassle) [reply CHANGELOG_PENDING.md:8]: We should probably use changie to stop cases like this....

Comment (justinvp) [reply CHANGELOG_PENDING.md:8]: > We should probably use changie to stop cases like this....

Yes

**Pull request merged**